model/team_member.go

```diff
@@ -44,6 +44,13 @@ type EmailInviteWithError struct {
 	Error *AppError `json:"error"`
 }
 
+type TeamMembersGetOptions struct {
+	// Sorting option
+	Sort string
+	// if true, then Users.DeleteAt == 0
```

Comment:
Having trouble parsing this, should we just say:
```
	// If true, exclude team members whose corresponding user is deleted.
```

Reply:
Changed! Thanks for the suggestion.